ansible/playbook/openstack/openstack_auth_passstore_v3password.yml

```diff
@@ -4,13 +4,30 @@
   gather_facts: false
 
   tasks:
-    - name: "Set openstack_auth facts"
-      set_fact:
-        openstack_auth: 
-          openstack_project_name: "{{ query('passwordstore', 'openstack/host/project_name')[0] }}"
-          openstack_console_user:  "{{ query('passwordstore', 'openstack/host/console_user')[0] }}"
-          openstack_console_password: "{{ query('passwordstore', 'openstack/host/console_pw')[0] }}"
-          openstack_user_domain:  "{{ query('passwordstore', 'openstack/host/console_domain')[0] }}"
-          openstack_project_domain: "{{ query('passwordstore', 'openstack/host/os_domain')[0] }}"
-          openstack_os_auth_url: "{{ query('passwordstore', 'openstack/host/os_auth_url')[0] }}"
+    # - name: "Set openstack_auth facts"
+    #   ansible.builtin.set_fact:
+    #     openstack_auth:
+    #       openstack_project_name: "{{ query('passwordstore', 'openstack/host/project_name')[0] }}"
+    #       openstack_console_user: "{{ query('passwordstore', 'openstack/host/console_user')[0] }}"
+    #       openstack_console_password: "{{ query('passwordstore', 'openstack/host/console_pw')[0] }}"
+    #       openstack_user_domain: "{{ query('passwordstore', 'openstack/host/console_domain')[0] }}"
+    #       openstack_project_domain: "{{ query('passwordstore', 'openstack/host/os_domain')[0] }}"
+    #       openstack_os_auth_url: "{{ query('passwordstore', 'openstack/host/os_auth_url')[0] }}"
+
+    - name: "Set authentication vars"
+      ansible.builtin.set_fact:
+        rhos_auth:
+          auth_url: "{{ query('passwordstore', 'openstack/host/os_auth_url')[0] }}"
+          password: "{{ query('passwordstore', 'openstack/host/console_pw')[0] }}"
```

Comment:
Is it described somewhere that passwordstore is needed/mandatory ?

Reply:
Yes, on the OpenStack playbook README. This playbook is to be used with the passwordstore to store the results of the installation process and the access information.

https://github.com/jacobdotcosta/k8s-infra/tree/remove-cloud-infra-collection/ansible/playbook/openstack#readme